test/controllers/categories_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
```

Comment:
On your controller tests you should also include some negative cases, what if the user tries to see a category that's been deleted or never existed.  What if someone without permission tries to edit a category?